### ISSUE TITLE
Add error summary component

### ISSUE THREAD

**Issue by maya** [uswds/uswds#4019]
**Is your feature request related to a problem? Please describe.**
When someone makes more than one error in a form, even though gov.uk says to show it for just one.

**Describe the solution you'd like**
https://design-system.service.gov.uk/components/error-summary/

**Comment by thisisdano** [comment]
In lieu of more specialized component, would an alert box placed at the top of the form do the trick? (In addition to error helper text in the relevant fields.)

**Comment by maya** [comment]
Yes, I'm thinking USWDS error box + GOV.UK content and functionality.

**Comment by danizen** [comment]
Is this not already possible with clever coding?

**Comment by amycole501** [comment]
Adding this to consideration pending an a11y and UX audit; part of our work to evaluate best practices and examples of how to provide error prevention, suggestion and display guidance.

**Comment by annepetersen** [comment]
Hi there: this is one of many submitted PRs/issues that have languished here. USWDS is currently understaffed, so I'm closing items that aren’t as high on the priority list right now, including this one, unfortunately.
